bindings/python/algorithm/expose-model.cpp

```diff
@@ -2,17 +2,19 @@
 // Copyright (c) 2019-2020 INRIA
 //
 
+#include "pinocchio/algorithm/model.hpp"
```

Comment:
Please do not change the include order for no reason

Reply:
Sorry, my editor is doing more than I know.